### DISCUSSION TITLE
regrid of  climo grid data

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1404] [Configuration] [answered]
I have a question for regrid of climo data grid in ensemble stat  verification:
My application will use 1 degree grid GEFS data , 1 degree grid GFS analysis and 1.5 degree  EVA climo data.  The verification will be over 1 degree global domain. This means only climo 1.5 degree data nedd to be regrided to 1 degree data. In my conf file, I am trying to use   GRID_STAT_CLIMO_MEAN_REGRID_TO_GRID and GRID_STAT_CLIMO_STDEV_REGRID_TO_GRID to regrid both clima mean and standard deveation data to 1 degree data. But I can not find these 2 conf variables in the METplus
document.  So how to make this regrinding? I can find GRID_STAT_REGRID_TO_GRID
and ENSEMBLE_STAT_REGRID_TO_GRID, but no CLIMO regrid. 

Binbin Zhou
EMC/NCEP

**Comment by georgemccabe** [comment, marked as answer]
Hi @BinbinZhou-NOAA,

The climo_mean.regrid and climo_stdev.regrid dictionaries do not support a "to_grid" value. The climatology data is automatically regridded to the verification grid, so you do not need to explicitly specify it.

**Reply by BinbinZhou-NOAA** [reply]
@georgemccabe 
George,

 OK,  Thanks!

Binbin
